### PR TITLE
Remove unused variables

### PR DESCRIPTION
**Pull request by vtomole**

### CODE REVIEW

**Comment by vtomole** [comment]
@kevinsung How do we address the coverage check here?

File: src/openfermion/measurements/_equality_constraint_projection_test.py [hunk 1]
```diff
@@ -55,14 +54,14 @@ def setUp(self):
 
     def test_linearize_term(self):
         past_terms = set()
-        for term, coefficient in self.fermion_hamiltonian.terms.items():
+        for term, __ in self.fermion_hamiltonian.terms.items():
```
Comment (kevinsung) [inline src/openfermion/measurements/_equality_constraint_projection_test.py]: Can we configure your tool to use just one underscore `_` instead of two?

**Comment by kevinsung** [comment]
Let's just override the coverage check in this case. The longer term solution is #420 .

File: src/openfermion/measurements/_equality_constraint_projection_test.py (hunk 1 above)
Comment (vtomole) [reply src/openfermion/measurements/_equality_constraint_projection_test.py]: Done

**Comment by vtomole** [comment]
@kevinsung Ready for merging.

File: .travis.yml [hunk 2]
```diff
@@ -58,5 +58,5 @@ install:
 script: sphinx-build -nW -b html -d docs/_build/doctrees/ docs/ docs/_build/html
 script: export OMP_NUM_THREADS=1 && pytest src/openfermion --cov src/openfermion
 
-after_success:
-  - coveralls
+# after_success:
+#   - coveralls
```
Comment (kevinsung) [inline .travis.yml]: Oh I didn't mean to actually disable the check. I meant that we could merge this PR in spite of the failing check.

Comment (vtomole) [reply .travis.yml]: I didn't know that was possible. I thought you had to disable, merge then re-enable. I've reverted this.

**Pull request merged**